bundles/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/protocol/commandclass/ZWaveMultiInstanceCommandClass.java

```diff
@@ -353,34 +334,7 @@ private void handleMultiChannelCapabilityReportResponse(SerialMessage serialMess
 			}
 
 			// Add all the command classes supported by this endpoint
-			for (int i = 0; i < serialMessage.getMessagePayload().length - offset - 3; i++) {
```

Comment:
The code is moved (extracted). To improve readability. No functional change in this commit.